test/Microsoft.Azure.SignalR.IntegrationTests/MessageOrderTests.cs

```diff
@@ -98,35 +99,35 @@ public async Task OutgoingMessagesUseSameServiceConnection()
                     // finally, get ready to verify the order of messages
                     int actualCallbackNum = (int)invMsg.Arguments[0];
 
-                    if (MessageOrderFixed)
-                    {
-                        // this check works for both primary and secondary connections
-                        Assert.Equal(actualCallbackNum, msgCount);
-                    }
+                    // TODO: unreable code
```

Comment:
?

Reply:
Fix some build warnings. The part of code is not working now.

Reply:
reverted